probe/elasticsearch.go

```diff
@@ -428,14 +543,46 @@ func (es *EsProbe) indexDocument(index, documentID string, esDoc *EsDocument) (f
 	return durationMilliSec, nil
 }
 
-func (es *EsProbe) createMissingIndex(index string) error {
+func (es *EsProbe) isIndexExist(index string) (bool, error) {
```

Comment:
why not something like: "indexExists"? / Just a suggestions because the name looks weird to me